src/commands/registryCredentials/dockerLogin/DockerLoginRegistryCredentialsContext.ts

```diff
@@ -17,4 +17,6 @@ export interface DockerLoginRegistryCredentialsContext extends CreateAcrContext,
 
     newRegistrySecret?: Secret;
     newRegistryCredential?: RegistryCredentials;
+
+    suppressEnableAdminUserPrompt?: boolean;
```

Comment:
Can't this just check `suppressConfirmation`?

Reply:
I don't think that prop exists on the context currently.  There is the `suppressNotification` option in user settings for resources, but I don't think we want to use that because this is ordinarily a prompt that I would always want to show.  `suppressConfirmation` in this case is an explicit option that is being passed into the `deployWorkspaceProjectInternal` command because an outside extension specifically wants to bypass.